docs/source/models/concept_models/vivarium_nutrition_optimization/pregnancies/concept_model.rst

```diff
@@ -588,6 +588,22 @@ Specific outputs for specific models are specified in the following section.
         * Hemorrhage effect on postpartum hemoglobin effects are as expected (assessed via interactive sim)
         * Hemoglobin effect on maternal hemorrhage incidence is as expected. Maternal hemorrhage incidence still verifies as the population level. Note that there was a resolved bug where maternal *disorders* PAFs and RRs were applied to maternal hemorrhage, but this was resolved.
         * Hemoglobin on maternal disorders PAFs and RRs applied as expected, however, maternal disorders incidence (and therefore mortality) are slightly underestimated at the population level. This is due to risk-affected probabilities of an incident maternal disorder case greater than 1 for a substantial number of simulants with low hemoglobin levels. More details discussed in table below.
+  * - 5.0
+    - Verify that joint anemia/BMI risk exposure matches expected value and that exposure does not change over time with changing hemoglobin
+    - Looks great! 
+        * `Model 5.0 anemia exposure <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_5.0_maternal_disorders_anemia.ipynb>`_
+        * `Model 5.0 joint anemia/BMI exposure prevalence <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_5_bmi_exposure.ipynb>`_
+        * `Interactive sim to check that joint anemia/BMI exposure does not change over time <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_7_interactive_sim.ipynb>`_
+  * - 6.0
+    - Verify:
+        * Scenario-specific intervention coverage
+        * Intervention impacts on hemoglobin exposure
+    - * `Intervention coverage looks good by scenario <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_6_intervention_coverage.ipynb>`_, and `confirmed to be appropriately targeted to BMI exposure in the interactive sim <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/verification_and_validation/pregnancy_model/model_7_interactive_sim.ipynb>`_
+      * Draw-level uncertainty in intervention effect on hemoglobin erroneously applied as individual-level stochastic uncertainty, as shown in the interactive sim linked above
```

Comment:
The logic makes sense here--the reason I sampled this at the individual level is that is what IV Iron was doing, unless I'm understanding the code incorrectly. Just added this in the (forthcoming) bugfix.

Reply:
This is looking good in the latest run, removed the V&V issue in the latest commit